holmes-py/specs/gdc_data_portal_v2/data_release/40/cases_inclusion.spec

```diff
@@ -0,0 +1,36 @@
+# Data Release: 40 - Cases Inclusion
+Date Created        : 06/03/2024
+Version			    : 1.0
+Owner		        : GDC QA
+Description		    : Cases Introduced in DR-40
+Test-Case           : PEAR-1929
+
+tags: gdc-data-portal-v2, data-release
+
+## Case Introduced in this Data Release: C3L-04090
+* On GDC Data Portal V2 app
+* Quick search for "9cc09304-d998-4697-a38a-8b02325974a6" and go to its page
+* Add all files to cart on the case summary page
+* Is text "Added 39 files to the cart." present on the page
+* Remove all files from cart on the case summary page
+* Is text "Removed 39 files from the cart." present on the page
+* Verify the table "Summary Case Summary" is displaying this information
+    |text_to_validate                       |
+    |---------------------------------------|
+    |9cc09304-d998-4697-a38a-8b02325974a6   |
+    |CPTAC-3                                |
+    |Gliomas                                |
+
+## Case Introduced in this Data Release: MATCH-S1-B5KS
+* On GDC Data Portal V2 app
+* Quick search for "740a2d74-2e3c-4d81-9a4d-2b750306612a" and go to its page
+* Add all files to cart on the case summary page
+* Is text "Added 18 files to the cart." present on the page
+* Remove all files from cart on the case summary page
+* Is text "Removed 18 files from the cart." present on the page
+* Verify the table "Summary Case Summary" is displaying this information
+    |text_to_validate                       |
+    |---------------------------------------|
+    |740a2d74-2e3c-4d81-9a4d-2b750306612a   |
```

Comment:
each row generally denotes a new dataset. What are your thoughts about having them horizontally aligned? (just a discussion, not recommending to make any changes)

Reply:
I prefer it being in a column. Some tables are quite big with many checks, and if that went to the right instead of down would be much harder to read